pkg/clusteroperator/clusteroperators.go

```diff
@@ -0,0 +1,273 @@
+package clusteroperator
+
+import (
+	"context"
+	"time"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	v1 "github.com/openshift/api/config/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"fmt"
+)
+
+// CoListBuilder provides struct for ClusterOperator List.
+type CoListBuilder struct {
+	Definition *v1.ClusterOperatorList
```

Comment:
Can you please add comments to all public elements of struct. Just to be consistent with other packages.

Reply:
added